src/gtk/menu.cpp

```diff
@@ -762,6 +766,36 @@ void wxMenuItem::SetBitmap(const wxBitmap& bitmap)
     }
 }
 
+wxBitmap wxMenuItem::GetBitmap() const
```

Comment:
This should ideally reuse the static `wxBitmapBundle::GetBitmapFor()` overload mentioned in another PR discussion.